Products/DataCollector/zenmodeler.py

```diff
@@ -1110,10 +1115,10 @@ def mainLoop(self, driver):
         if self.clients:
             self.log.error("Modeling cycle taking too long")
             return
-
+        self.didCollect = False
```

Comment:
Is it necessary to set this if it's already set in the class instantiation where you declare `self.didCollect = False` (line 117)?